packages/project-account-switcher/src/presenters/ProjectAccountSwitcherPresenter.js

```diff
@@ -81,67 +79,59 @@ export default class ProjectAccountSwitcherPresenter extends Component {
   constructLabelPlaceholder() {
     const placeholders = [];
     if (this.props.activeAccount) {
-      placeholders.push(
-        ProjectAccountSwitcherPresenter.constructPlaceholder(
-          this.props.activeAccount.label
-        )
-      );
+      placeholders.push(constructPlaceholder(this.props.activeAccount.label));
     }
     if (this.props.activeProject) {
-      placeholders.push(
-        ProjectAccountSwitcherPresenter.constructPlaceholder(
-          this.props.activeProject.label
-        )
-      );
+      placeholders.push(constructPlaceholder(this.props.activeProject.label));
     }
     return placeholders.join("/");
   }
 
   accountsList() {
-    return this.props.accounts.map(account => (
+    return this.props.accounts.map(({ id, label }) => (
```

Comment:
`accountsList()` and `projectsList()` seem to do the same thing except for some markup changes (accounts/project) and the `onClick` event, wondering would it make sense to combine the two functions, and use some conditional logic to switch from rendering accounts or projects?

Reply:
I agree, though the only thing I'm trying to do here is fix the branch. There are quite a few functions that can made reusable in this package.